filters/special_casing/filter.py

```diff
@@ -0,0 +1,21 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class TextSpecialCasingFilter(SentenceOperation):
+    """Filter if any of the special casings appear in the sentence:
+    All text is capitalized or All text is lowercased or there is Title Casing in the Sentence.
+    """
+
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION, TaskType.TEXT_TAGGING, TaskType.QUESTION_ANSWERING, TaskType.QUESTION_GENERATION, TaskType.SENTIMENT_ANALYSIS]
+    languages = ["en", "fr", "es", "it", "pt", "fi", "nl", "da", "cs", "hr", "bg", "be", "eu", "ru", "uk", "pl", "sv", "sk", "sl"]
+    keywords = ["casing", "uppercase", "lowercase", "titlecase", "Latin", "Cyrillic"]
```

Comment:
Seems you've added some new keywords here and was contemplating if all should be a part of the keywords list in docs. Some of them are very specific for your transformation and I also think that can be helpful as general keywords (for search, etc) rather than only for classification. What do you think @mille-s ?